package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "promise-android-tools",
-  "version": "4.0.13",
+  "version": "4.0.14",
```

Comment:
Are we supposed to bump version anytime after a release or only when releasing?

I think the first option causes confusion when going through git history.